lib/recipes/import/taste_of_home.rb

```diff
@@ -43,8 +51,8 @@ def recipe_directions
 
       def total_time
         document
-          .css('div.total-time p')
-          .text
+          .css('div.recipe-time span')
+          .map(&:text)
       end
     end
   end
```

Comment:
Doesn't look like this is being used anymore, so I'd just delete it.

**edit**: Actually I guess your code is using it, but my suggestion does not use it.